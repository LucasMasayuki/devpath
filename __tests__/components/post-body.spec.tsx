import React from 'react'
import renderer from 'react-test-renderer'
import PostBody from '@/components/post-body'

describe('PostBody', () => {
  it('should render PostBody', () => {
    const component = renderer.create(<PostBody content="<div>test</div>" />)

    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
