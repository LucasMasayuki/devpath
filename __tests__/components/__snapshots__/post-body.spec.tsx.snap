// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostBody should render PostBody 1`] = `
<div
  className="css-0"
>
  <div
    class="markdown"
    className="css-0"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<div>test</div>",
      }
    }
  />
</div>
`;
